packages/bigtable/src/index.js

```diff
@@ -262,267 +279,262 @@ function Bigtable(options) {
     return new Bigtable(options);
   }
 
-  if (!options.cluster) {
-    throw new Error('A cluster must be provided to interact with Bigtable.');
-  }
-
-  if (!options.zone) {
-    throw new Error('A zone must be provided to interact with Bigtable.');
-  }
-
-  options = extend({}, options, {
-    zone: options.zone.name || options.zone
-  });
-
-  this.clusterName = format(
-    'projects/{projectId}/zones/{zone}/clusters/{cluster}',
-    options
-  );
+  var adminBaseUrl = 'bigtableadmin.googleapis.com';
 
   var config = {
     baseUrl: 'bigtable.googleapis.com',
     service: 'bigtable',
-    apiVersion: 'v1',
+    apiVersion: 'v2',
     protoServices: {
-      BigtableService: googleProtoFiles.bigtable.v1,
-      BigtableTableService: {
-        path: googleProtoFiles.bigtable.admin,
-        service: 'bigtable.admin.table'
+      Bigtable: googleProtoFiles.bigtable.v2,
```

Comment:
Since the common module uses the latest version of google-proto-files now, is there a way to pass some kind of config here without requiring a dependency on the module in Bigtable?